Intersect.Client.Framework/Gwen/Control/ScrollControl.cs

```diff
@@ -247,6 +247,27 @@ protected override bool OnMouseWheeled(int delta)
             return false;
         }
 
+        /// <summary>
+        ///     Handler invoked on mouse wheel event.
+        /// </summary>
+        /// <param name="delta">Scroll delta.</param>
+        /// <returns></returns>
+        protected override bool OnMouseHWheeled(int delta)
+        {
+
+            if (CanScrollH && mHorizontalScrollBar.IsVisible)
+            {
+                if (mHorizontalScrollBar.SetScrollAmount(
+                    mHorizontalScrollBar.ScrollAmount - mHorizontalScrollBar.NudgeAmount * (delta / 60.0f), true
```

Comment:
Can you break this calculation into a separate line

Reply:
No proble, just changed!